lib/v2/screens/dashboard/wallet_screen.dart

```diff
@@ -101,7 +101,11 @@ class _WalletScreenState extends State<WalletScreen> with AutomaticKeepAliveClie
 
   Widget walletBottom(BuildContext context) {
```

Comment:
on a different pr, pls move this `walletBottom` to a component and to its own class

Reply:
Ok!